SU-SWS/stanford-jumpstart-deployer
# Security updates for March, 2019

# READY FOR REVIEW

# Summary
- Security updates

# Needed By (Date)
- Next week sometime is fine. Let's say March 7th.

# Criticality
- How critical is this PR on a 1-10 scale? 3/10
- Regular maintenance, nothing critical I believe

# Steps to Test

1. Check out this branch
2. Run:
```
  dr7 make --concurrency=4 make/production/jumpstart-academic.make /tmp/jsa-prod
  dr7 make --concurrency=4 make/production/jumpstart-engineering.make /tmp/jse-prod
  dr7 make --concurrency=4 make/production/jumpstart-plus.make /tmp/jsplus-prod
  dr7 make --concurrency=4 make/production/jumpstart.make /tmp/jsv-prod
```
3. Ensure that all 4 products build correctly

# Affected Projects or Products
- Anchorage

# Associated Issues and/or People
- Anyone who should be notified? ( @sherakama )

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
